src/main/java/dk/kb/util/BuildInfoManager.java

```diff
@@ -108,5 +152,10 @@ private static synchronized void loadBuildInfo() {
         name = properties.getProperty("APPLICATION.NAME", "unknown");
         version = properties.getProperty("APPLICATION.VERSION", "unknown");
         buildTime = properties.getProperty("APPLICATION.BUILDTIME", "unknown");
+        gitCommitChecksum = properties.getProperty("GIT.COMMIT.CHECKSUM", "unknown");
+        gitBranch = properties.getProperty("GIT.BRANCH", "unknown");
+        gitClosestTag = properties.getProperty("GIT.CLOSEST.TAG", "unknown");
```

Comment:
Closest git tag is stated as `${git.tag}` if it cannot be resolved. The properties loader should mark the key as `unknown`in this situation. In principle this goes for all the properties.

Reply:
Points for handling it during build, so that the generated file is "correct", instead of handling it at access time.